### PR TITLE
Added youtube-dlc

### PR DESCRIPTION
**Pull request by chriscn**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----
`youtube-dlc` is the new version of `youtube-dl` as the old version has becoome stagant. See this [Reddit](https://www.reddit.com/r/DataHoarder/comments/ir8ic6/youtubedlc_an_active_fork_of_youtubedl/) post for more information.

### CODE REVIEW

File: Formula/youtube-dlc.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+class YoutubeDlc < Formula
+  desc "Download YouTube videos from the command-line"
+  homepage "https://blackjack4494.github.io/youtube-dlc/"
+  version "2020.09.13"
+  url "https://github.com/blackjack4494/youtube-dlc/releases/download/#{version}/youtube-dlc"
```
Comment (SMillerDev) [inline Formula/youtube-dlc.rb:5]: https://docs.brew.sh/Acceptable-Formulae#we-dont-like-binary-formulae

File: Formula/youtube-dlc.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  sha256 "633caf38c990ab1b21cfe849151601e8a1548ddcdd539caf755e1b3d47941af9"
+  license "Unlicense"
+
+  def install
+    bin.install "youtube-dlc"
+  end
+
+  test do
+    # commit history of homebrew-core repo
```
Comment (SMillerDev) [inline Formula/youtube-dlc.rb:14]: ?

**Comment by SMillerDev** [comment]
> youtube-dlc is the new version of youtube-dl as the old version has becoome stagant.

Seeing how they made a release today I'm not sure if it's stagnant, but that shouldn't block this PR.

File: Formula/youtube-dlc.rb (hunk 2 above)
Comment (kristofferR) [reply Formula/youtube-dlc.rb:14]: That weird comment is also frankly in the youtube-dl.rb formula that was modified in this PR.

**Pull request closed without merging**